docs/sources/http_api/serviceaccount.md

```diff
@@ -0,0 +1,341 @@
++++
+title = "service account HTTP API "
+description = "Grafana service account HTTP API"
+keywords = ["grafana", "http", "documentation", "api", "serviceaccount"]
+aliases = ["/docs/grafana/latest/http_api/serviceaccount/"]
++++
+
+# Service account API
+
+> If you are running Grafana Enterprise and have [Fine-grained access control]({{< relref "../enterprise/access-control/_index.md" >}}) enabled, for some endpoints you would need to have relevant permissions.
```

Comment:
I know that you didn't write this intro. Just as a general rule for teaching sake :), we don't start out a topic with a Note.

Maybe we could take this opportunity to change this. Maybe add an intro sentence like "You can use the service account API to create, edit, and delete service accounts." Just a thought.